src/VisualStudio/Core/Def/Microsoft.VisualStudio.LanguageServices.nuspec

```diff
@@ -23,5 +23,6 @@
       https://docs.microsoft.com/en-us/nuget/create-packages/select-assemblies-referenced-by-projects.
     -->
     <file src="net472\Microsoft.VisualStudio.LanguageServices.dll" target="lib\net472" />
+    <file src="net472\Microsoft.VisualStudio.LanguageServices*.pdb" target="lib\net472" />
```

Comment:
Are these being included now because they're portable PDB hence acceptable size increase?

Reply:
Yes. We already include PDBs in all our packages. It's the de facto standard now to do so.